posthog/tasks/usage_report.py

```diff
@@ -940,9 +940,9 @@ def get_teams_with_rows_exported_in_period(begin: datetime, end: datetime) -> li
             finished_at__gte=begin,
             finished_at__lte=end,
             status=BatchExportRun.Status.COMPLETED,
-            batch_export__model=BatchExport.Model.EVENTS,
             batch_export__deleted=False,
```

Comment:
Do you know why this filter is present? Just thinking we could be missing runs executed before a batch export being deleted. Or do you think we don't need to worry about this?

Reply:
> Just thinking we could be missing runs executed before a batch export being deleted. Or do you think we don't need to worry about this?

How could this happen? Also, why would _removing_ a filter cause us to _miss_ runs? If anything we should be capturing _more_ runs than with the filter?

Reply:
Oh, no clue about that one, I haven't considered it. I guess we need to chat a bit about it.

Reply:
ok, might be a bit of an edge case but won't let it block this PR